addons/godot-next/2d/trail_2d.gd

```diff
@@ -79,8 +80,8 @@ func _notification(p_what: int):
 			self.target_path = @""
 			self.trail_length = 0
 
-#warning-ignore:unused_argument
-func _process(delta: float):
+
```

Comment:
Is there a reason this warning ignore was removed?

Reply:
Variables declared with an underscore are ignored automatically, so there's no real need to have an entire warning exemption for it in this case.